core/src/main/java/org/pentaho/platform/dataaccess/datasource/wizard/public/datasourceEditorDialog.css

```diff
@@ -248,10 +248,6 @@ aggregation-dialog-checkbox-label {
   height: 350px !important;
 }
 
-#database-options-box input {
-  width: 260px !important;
-}
```

Comment:
This css file is not exclusive to the `ruby` theme, so this selector should not be removed.
You can override it with something like this:
```
#database-options-box input {
  width: 260px !important;
}
:where(.responsive) #database-options-box input {
  width: auto !important;
}
```